### ISSUE TITLE
Unexpected argument warning during csv import

### ISSUE THREAD

**Issue by chiatt** [archesproject/arches-for-science-prj#956]
The following may be caused my a function method that should take a `**kwargs` argument to accept 'context'
```
2022-09-02 14:12:09,349 arches.app.models.tile WARNING  No associated functions or other TypeError raised by a function
2022-09-02 14:12:09,351 arches.app.models.tile WARNING  post_save() got an unexpected keyword argument 'context'
```

**Comment by chiatt** [comment]
This may be resolved through the removal of unused functions: https://github.com/archesproject/arches-for-science-prj/pull/912/files

**Comment by chiatt** [comment]
Resolved by #912
